lib/services/image-service.ts

```diff
@@ -0,0 +1,86 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+import temp = require("temp");
+
+class ImageService implements IImageService {
+	private replaceAll: boolean;
+
+	constructor(private $devicePlatformsConstants: Mobile.IDevicePlatformsConstants,
+		private $errors: IErrors,
+		private $fs: IFileSystem,
+		private $logger: ILogger,
+		private $progressIndicator: IProgressIndicator,
+		private $projectConstants: Project.IProjectConstants,
+		private $project: Project.IProject,
+		private $prompter: IPrompter,
+		private $server: Server.IServer) {
+
+		this.$project.ensureProject();
+
+		if (!this.$project.capabilities.imageGeneration) {
+			this.$errors.failWithoutHelp("This command is not applicable to %s projects ", this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebsite);
+		}
```

Comment:
`this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebsite` should be `this.$project.projectData.Framework`